resources/js/app-layout.js

```diff
@@ -215,6 +224,10 @@ window.ProcessMaker.apiClient.interceptors.response.use((response) => {
   // response.config.method (PUT, POST, DELETE)
   // response.config.url (extract resource name)
   window.ProcessMaker.EventBus.$emit("api-client-done", response);
+  //flags print forms
+  if (window.ProcessMaker.apiClient.requestCountFlag) {
+    window.ProcessMaker.apiClient.requestCount--;
```

Comment:
It looks like requestCount can have negative numbers. To avoid problems do not decrease it if it is 0